index.html

```diff
@@ -296,141 +292,176 @@
     var daySpan = document.getElementById("day");
     daySpan.firstChild.textContent = dateToString(dateObj);
   }
-  function renderSchedule(dateObj, schedule, type, scheduleElement) {
-    var school;
-    if (schedule.grade <= 8) {
-      school = "MS";
+  function getFunnyImageObj(dayEventObj) {
+    var numImages = Math.floor((Math.random() * dayEventObj.imageNum) + 1);
+    var imageUrl = "/images/" + dayEventObj.name + numImages.toString() + ".jpg";
+    var imageText = dayEventObj.imagePrefix;
+    return {url: imageUrl, text: imageText};
+  }
+
+  // Determine whether the user is in middle or upper school
+  function getSchool(grade) {
+    if (grade <= 8) {
+      return "MS";
     } else {
-      school = "US";
+      return "US";
     }
+  }
 
-    var todaySchedule = [];
-    var scheduleObj;
+  // Handles exceptions, also handles special weekend schedules
+  function getDaySchedule(dateObj, exceptions) {
```

Comment:
Name of this function is strange, since it doesn't return a schedule. Maybe getScheduleTypeForDate?